sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1744,14 +1744,14 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
 
   val SHUFFLE_COMPRESSION_CODEC = conf("spark.rapids.shuffle.compression.codec")
     .doc("The GPU codec used to compress shuffle data when using RAPIDS shuffle. " +
-      "Supported codecs: lz4, copy, none")
+      "Supported codecs: zstd, lz4, copy, none")
     .internal()
     .startupOnly()
     .stringConf
     .createWithDefault("none")
 
-val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.lz4.chunkSize")
```

Comment:
Do we want to combine these?  If we do, we cannot create separate defaults between LZ4 and ZSTD if it's determined different values are better defaults for one codec vs another.  Similarly, consider if we supported a compression level.  Compression level 4 might mean something very different for one codec vs. another, or not apply at all despite the config implying it does.

I'm not sure myself which way to go on this, just musing the potential confusion in the future if we try to combine.

Reply:
Yeah, you are right. Better to use separate configs for LZ4 and ZSTD. Updated